keyboards/moondrop/dash75/keymaps/via/keymap.json

```
@@ -0,0 +1,32 @@
+{
+    "version": 1,
+    "notes": "This file is a keymap.json file for moondrop/dash75",
+    "keyboard": "moondrop/dash75",
+    "keymap": "via",
+    "layout": "LAYOUT",
+    "author": "vinorodrigues",
+    "config": {
+        "features": {
+            "via": true,
+            "lto": true
+        }
+    },
+    "layers": [
+        [
+            "KC_ESC",  "KC_F1",   "KC_F2",   "KC_F3",   "KC_F4",              "KC_F5",   "KC_F6",   "KC_F7",   "KC_F8",   "KC_F9",   "KC_F10",  "KC_F11", "KC_F12",   "KC_PSCR",
+            "KC_GRV",  "KC_1",    "KC_2",    "KC_3",    "KC_4",    "KC_5",    "KC_6",    "KC_7",    "KC_8",    "KC_9",    "KC_0",    "KC_MINS", "KC_EQL", "KC_BSPC",  "KC_INS",
+            "KC_TAB",  "KC_Q",    "KC_W",    "KC_E",    "KC_R",    "KC_T",    "KC_Y",    "KC_U",    "KC_I",    "KC_O",    "KC_P",    "KC_LBRC", "KC_RBRC", "KC_BSLS", "KC_DEL",
+            "KC_CAPS", "KC_A",    "KC_S",    "KC_D",    "KC_F",    "KC_G",    "KC_H",    "KC_J",    "KC_K",    "KC_L",    "KC_SCLN", "KC_QUOT",            "KC_ENT",  "KC_PGUP",
+            "KC_LSFT", "KC_Z",    "KC_X",    "KC_C",    "KC_V",    "KC_B",    "KC_N",    "KC_M",    "KC_COMM", "KC_DOT",  "KC_SLSH",            "KC_RSFT", "KC_UP",   "KC_PGDN",
+            "KC_LCTL", "KC_LGUI", "KC_LALT",                       "KC_SPC",                                   "KC_RALT", "KC_APP",             "KC_LEFT", "KC_DOWN", "KC_RGHT"
+        ],
+        [
+            "_______", "_______", "_______", "_______", "_______",            "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______",
+            "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______",
+            "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______",
+            "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______",            "_______", "_______",
+            "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______",            "_______", "_______", "_______",
+            "_______", "_______", "_______",                       "_______",                                  "_______", "_______",            "_______", "_______", "_______"
+        ]
```

No point including layers that cannot be accessed
```suggestion
        ]
```

---

* the blanks on the `via` keymap are intentional ... in VIA one can map keys post initial population of the dynamic eeprom allocation.  this allows for the user to add their own "Fn" functionality if they want.
* agree that it should go from the `default` keymap.

---

You are incorrect in this being a requirement, it hasnt been for quite a while.

https://github.com/qmk/qmk_firmware/blob/master/quantum/dynamic_keymap.c#L158

This is why the layers can be removed, and why it is being requested.

---

ahhh... right.  removing.